timers.js

```diff
@@ -1,4 +1,28 @@
+var start = $("#start");
+var reset = $("#reset");
+var pause = $("#pause");
 
+function countUp(){
+  ++seconds;
+$("h1").text("Time Lapse: " + seconds);
+}
 
+var seconds = 0;
+function startTimer() {
+  $("h1").text("Time Lapse:" + seconds);
 
+  var timer = setInterval(countUp,1000);
 
+  pause.on("click", function(){
+    clearInterval(timer);
+  })
+  reset.on("click", function(){
+    clearInterval(timer);
+    seconds = 0;
+    $("h1").text("Stop Watch");
+  })
```

Comment:
Put the `pause` and `reset` click events outside of `startTimer()`. You virtually never want to have a `click` event defined inside another `click` event.

For instance, if you have this:

```html
<button id="test1"></button>
<button id="test2"></button>
```

```js
$("#test1").on("click", function(){
  $("#test2").on("click", function(){
    console.log("hello");
  });
});
```

If you click on `test1` 10 times, then click on `test2`, you'll get 10 `hello` in your console. That's because each time you click on `test` it'll add a new instance of something saying, "OK, when `test2` is clicked on, I should console log `hello`."